rocky/rocky/templates/oois/ooi_detail_origins_observations.html

```diff
@@ -24,7 +24,11 @@ <h2>{% translate "Last observed by" %}</h2>
                                     <a href="{% url 'boefje_detail' organization_code=organization.code plugin_id=observation.boefje.id %}"
                                        title="{{ observation.boefje.id }}">{{ observation.boefje.name }}</a>
                                 {% else %}
-                                    {% translate "The boefje has since been deleted or disabled." %}
+                                    {% if observation.normalizer.raw_data.boefje_meta.boefje.id == "manual" %}
+                                        {% translate "This scan was manually created." %}
```

Comment:
Not really blocking, but maybe "The raw file was uploaded manually" or "This scan was done manually" is better than "This scan was manually created"? In the case OOIs are just added manually they don't have to be the result of a scan, so I think saying that there was a manual upload instead of scan is more accurate.